edward/models/distributions.py

```diff
@@ -119,19 +157,31 @@ def entropy(self):
 
 class Distribution:
     """
-    Base class for distributions, p(x | params).
+    Base class for distributions
+
+    p(x | params) = prod_{idx in shape} p(x[idx] | params[idx])
+
+    where shape is the shape of x and params[idx] denote the
+    parameters of each random variable x[idx].
 
     Parameters
     ----------
-    num_factors : int
-        Number of factors.
+    shape : int, list, or tuple, optional
+        Shape of random variable(s). For multivariate distributions,
+        the outermost dimension denotes the multivariate dimension.
```

Comment:
also, should mention connection of last dimension and how it relates to `is_multivariate`